src/cpp/fastdds/xtypes/dynamic_types/DynamicTypeBuilderImpl.hpp

```diff
@@ -82,6 +85,9 @@ class DynamicTypeBuilderImpl : public traits<DynamicTypeBuilder>::base_type
 
     traits<DynamicType>::ref_type build() noexcept override;
 
+    ReturnCode_t copy_from(
+            traits<DynamicTypeImpl>::ref_type type);
+
 protected:
 
     traits<DynamicTypeBuilderImpl>::ref_type _this ();
```

Comment:
This method is not implemented. The same happens in DynamicDataImpl. Maybe they can be removed if they are not really being used in the implementation (even the ones defined in DynamicTypeImpl and DynamicTypeMemberImpl do not seem to be used).

Reply:
Done in c8b1745ba

Reply:
`DynamicDataImpl::_this` has not been implemented yet. Please, fix it

Reply:
Done in c8f6dde98